### PR TITLE
Fix for profile cleanup.

### LINKED ISSUE #58: [bug] clean profiles doesn't remove profiles
this is true (at least for configs/darwin.ini)

### PR DESCRIPTION
**Pull request by rpappalax**
r? @pdehaan 
Let me verify this on Windows server before mergen.

Closes #58

### CODE REVIEW

**Comment by rpappalax** [comment]
passing on Win Server:
 <servicesqa> Project _utils_daily_maintenance build #328: SUCCESS in 11 sec: https://services-qa-jenkins.stage.mozaws.net:8443/job/_utils_daily_maintenance/328/

File: fftool/firefox_profile.py [hunk 1]
```diff
@@ -57,7 +58,12 @@ def clean_profiles():
         os.remove(os.path.join(BASE_PROFILE_DIR, "profiles.ini"))
     except:
         pass
-    profile_dir = os.path.join(BASE_PROFILE_DIR, "Profiles")
+
+    if IniHandler.is_windows():
+        profile_dir = os.path.join(BASE_PROFILE_DIR, "Profiles")
+    else:
+        profile_dir = BASE_PROFILE_DIR
+
```
Comment (pdehaan) [inline fftool/firefox_profile.py]: I'm having some doubts. I "gut feel" like the `try..except` should also be wrapped in the `is_windows()` business. Will dig deeper.

Comment (pdehaan) [inline fftool/firefox_profile.py]: Actually, I take that back. This works on OSX, but I think has a bit of code smell to it.

For example, on OSX, if I do this:
```py
def clean_profiles():
    try:
        print("attempting to delete " + os.path.join(BASE_PROFILE_DIR, "profiles.ini"))
        os.remove(os.path.join(BASE_PROFILE_DIR, "profiles.ini"))
    except:
        pass
```

It prints:
```
attempting to delete _temp/profiles/profiles.ini
```  

I have a feeling that maybe the `try..except` profiles.ini business is Windows specific and maybe belongs in the `if` block.

Agree? Disagree? Haten mein gutsen?

**Comment by pdehaan** [comment]
Anyways, it works on Win/OSX (and we'll assume Linux).
Feel free to merge, and we can address the comments above in a future PR.

**Comment by rpappalax** [comment]
Agreed on all points. Actually I would like to replace the INI handler stuff throughout the tool with something simpler, but perhaps we can go ahead and merge this so that at least the clean profiles will work for our demo..  And I agree that we should remove the – –ALL option.  It's not really necessary even for our Jenkins, and adds clutter to the tool.
I also think we should consider revisiting the idea of moving all the OS specific path creation stuff off to an os_utils.py or something along those lines.

**Comment by pdehaan** [comment]
R+ 👍 

We can refactor in the future.

**Pull request merged**